game/scripts/npc/abilities/map.txt

```diff
@@ -104,6 +104,11 @@
 				"var_type" 				"FIELD_INTEGER"
 				"aura_bonus_armor"			"4"
 			}
+			"03"
+			{
+				"var_type" 				"FIELD_FLOAT"
+				"filling_bottle_timer"			"60.0"
```

Comment:
Something like `"bottle_refill_cooldown"` more suits it's meaning.
Also, remove `.0`